Shared/Classes/Controller/OTMapOptionsViewController.m

```diff
@@ -94,7 +94,7 @@ - (void)setupForPublicUser {
             atIndex:self.buttonIndex++ withIcon:@"heart"
           andAction:@selector(doCreateAction:)];
     
-    if (self.isPOIVisible) {
+    if (self.isPOIVisible) { // what is POI ?
         [self addOption:OTLocalizedString(@"propose_structure")
```

Comment:
@gregclermont A priori ce if ne sert plus à rien mais avant de le dégager à quoi ça correspond "POI" ?

Reply:
Ah ! C'est l'acronyme de [_point of interest_](https://fr.wikipedia.org/wiki/Point_d%27int%C3%A9r%C3%AAt_(topographie)), un terme jargon souvent utilisé dans les systèmes qui manipulent des données topographiques.

En l’occurrence, pour nous, ça désigne une adresse du guide de solidarité.